python/skupper_router_internal/management/agent.py

```diff
@@ -273,8 +273,13 @@ def __str__(self):
 
 
 class LogEntity(EntityAdapter):
-
     def __init__(self, agent, entity_type, attributes=None, validate=True):
+        # This is for backward compatibility, we will remove this after a few releases.
+        if attributes and attributes.get("enable") and "trace" in attributes.get("enable"):
+            enabled = attributes.get("enable")
+            if enabled:
```

Comment:
Is this check redundant? attributes.get("enabled") was already checked on line 278

Reply:
Yep. Fixed ! Thanks